### PR TITLE
Update Maps.py

### PR DESCRIPTION
**Pull request by sgkang**
Fix indexing bug for Wire map

### CODE REVIEW

**Comment by rowanc1** [comment]
Oh boy. Thanks @sgkang.

File: SimPEG/Maps.py [hunk 1]
```diff
@@ -354,7 +354,7 @@ def __init__(self, *args):
             wire = Projection(self.nP, slice(start, start + arg[1]))
             setattr(self, arg[0], wire)
             maps += [(arg[0], wire)]
-            start = arg[1]
+            start += arg[1]
```
Comment (lheagy) [inline SimPEG/Maps.py]: nice catch!

**Comment by sgkang** [comment]
No worries:0 
Here is what we could do those wires:0 Actually having fun with wires

image

(Current source (vector) inversion for SP problem)

**Comment by rowanc1** [comment]
Can you give a presentation soon to update people with what is going on!?

Looks awesome.

**Pull request merged**